src/ConfigFactory.php

```diff
@@ -110,6 +110,7 @@ public static function createConfig($localHome = null)
     public static function createConfigFromEnv($systemConfigEnv = null, $localConfigEnv = null)
     {
         $cacheDir = '/tmp';
```

Comment:
Properly would be better to remove caching all together in a follow-up pr.
This is only used by the GitHub adapter atm.